tests/queries/0_stateless/02493_allow_suspicious_indices.sql

```diff
@@ -0,0 +1,11 @@
+CREATE TABLE allow_suspicious_indices (id UInt32) ENGINE = MergeTree() order by (id, id) settings allow_suspicious_indices = 1;
+DROP TABLE IF EXISTS allow_suspicious_indices;
```

Comment:
```suggestion
DROP TABLE IF EXISTS allow_suspicious_indices;
CREATE TABLE allow_suspicious_indices (id UInt32) ENGINE = MergeTree() order by (id, id) settings allow_suspicious_indices = 1;
DROP TABLE IF EXISTS allow_suspicious_indices;
```
in case of invalid test before